build-system/tasks/performance/print-report.js

```diff
@@ -100,4 +100,40 @@ function printReport(urls) {
   });
 }
 
-module.exports = printReport;
+class PageMetrics {
```

Comment:
Could you add a comment above this class to briefly explain its purpose?

Reply:
Done.